lib/shipment_info.py

```diff
@@ -5,60 +5,85 @@
 import quopri
 from bs4 import BeautifulSoup
 from lib.objects_to_drive import ObjectsToDrive
-from typing import Any, Dict, Optional, TypeVar
-
-_T0 = TypeVar('_T0')
+from typing import Any, Dict, Optional, Union
 
 OUTPUT_FOLDER = "output"
-COSTS_FILENAME = "expected_costs.pickle"
-COSTS_FILE = OUTPUT_FOLDER + "/" + COSTS_FILENAME
+SHIPMENTS_FILENAME = "shipments.pickle"
+SHIPMENTS_FILE = OUTPUT_FOLDER + "/" + SHIPMENTS_FILENAME
+
+
+class OrderInfo:
+  """
+  A value class that stores the information associated with a given order.
+
+  Note that an order email can contain multiple orders within it if the order
+  is broken up into multiple sub-orders at the time it is placed.
+  """
+  def __init__(self, email_id: str, cost: float) -> None:
+    self.email_id = email_id
+    self.cost = cost
 
+  def __str__(self) -> str:
+    return f'email_id: {self.email_id}, cost: {self.cost}'
 
-class ExpectedCosts:
+class ShipmentInfo:
```

Comment:
as discussed, please rename this + the file

Reply:
Done.

Reply:
But, fun fact, this makes the existing shipments.pickle wrong because the module name for the serialized OrderInfo class is now wrong, so you'll need to download them all again.